tools/star.wdl

```diff
@@ -18,49 +16,54 @@ task build_star_db {
     }
 
     input {
-        Int ncpu = 1
         File reference_fasta
         File gtf
-        String stardb_dir_name = "star_db"
+        String stardb_name = "star_db"
+        Boolean use_all_cores = false
         Int memory_gb = 50
-        Int? disk_size_gb
+        Int modify_disk_size_gb = 0
+        Int ncpu = 1
         Int max_retries = 1
-        Boolean use_all_cores = false
     }
 
-    String stardb_out_name = stardb_dir_name + ".tar.gz"
+    String stardb_out_name = stardb_name + ".tar.gz"
     Float reference_fasta_size = size(reference_fasta, "GiB")
     Float gtf_size = size(gtf, "GiB")
-    Int disk_size = select_first([disk_size_gb, ceil(((reference_fasta_size + gtf_size) * 3) + 10)])
+    Int disk_size_gb = ceil(
+        ((reference_fasta_size + gtf_size) * 3) + 10
+    ) + modify_disk_size_gb
+
+    String memory_limit_bytes = (memory_gb - 2) + "000000000"
```

Comment:
```suggestion
    # Leave 2GB as system overhead
    String memory_limit_bytes = (memory_gb - 2) + "000000000"
```
I think a comment is useful here. Though I'm not sure this is the exact language to use.